lighthouse-core/test/audits/__snapshots__/metrics-test.js.snap

```diff
@@ -43,6 +43,48 @@ Object {
 }
 `;
 
+exports[`Performance: metrics evaluates valid input (with lcp) correctly 2`] = `
+Object {
+  "estimatedInputLatency": 1324,
+  "estimatedInputLatencyTs": undefined,
+  "firstCPUIdle": 14907,
+  "firstCPUIdleTs": 1671236823075,
+  "firstContentfulPaint": 4702,
+  "firstContentfulPaintTs": 1671226617803,
+  "firstMeaningfulPaint": 4702,
+  "firstMeaningfulPaintTs": 1671226617803,
+  "interactive": 14907,
+  "interactiveTs": 1671236823075,
+  "largestContentfulPaint": 15024,
```

Comment:
this is definitely concerning that it's *after* TTI, that seems like it should not be possible...

maybe we should be doing some math.max/math.min to make things make sense like we do for the other paint metrics?

Reply:
why would it not be possible? an image can take a while to load, and if the LCP element is an element, that delay is relevant. LCP has nothing to do with main thread in that case